packages/react-native-client/android/src/main/java/io/fishjam/reactnative/Errors.kt

```diff
@@ -11,8 +11,7 @@ class ConnectionError(
   reason: AuthError
 ) : CodedException(message = "Connection error: ${reason.error}")
 
-class MissingScreenSharePermission :
-  CodedException(message = "No permission to start screen share, call handleScreenSharePermission first.")
+class MissingScreenSharePermission : CodedException(message = "No permission to start screen share.")
```

Comment:
**nitpick:** 
Lets message be omitted in constructor.

Reply:
Why?

Reply:
It's shorter :P

Reply:
I like explicit parameters, especially when it's not obvious what it is. It's also consistent with how other errors are declared.

Reply:
From kotlin coding conventions:
```
Use the named argument syntax when a method takes multiple parameters of the same primitive type, or for parameters of Boolean type, unless the meaning of all parameters is absolutely clear from context.
```

`CodedException` takes two arguments which are trivial for exceptions : `message` and `cause`. Imo there is no point in redundant repeating parameter name.

Reply:
I see that consistency is more important, so it can be changed in future during some android refactoring.